pymatviz/ptable.py

```diff
@@ -463,8 +477,14 @@ def add_colorbar(
         # Add colorbar
         cbar_ax = self.fig.add_axes(coords)
 
+        # Set colorbar range
+        self._norm = Normalize(
+            vmin=cbar_range[0] or self.data.attrs["vmin"],
+            vmax=cbar_range[1] or self.data.attrs["vmax"],
```

Comment:
Need confirmation on this to ensure re-scale the normalizer would not have any side effect on value-color mapping.